webapp/store/app/actions/index.js

```diff
@@ -1,9 +1,63 @@
-export const ACTION_TYPE = {
-  updateCountries: 'app/countries/update',
-  updateRegions: 'app/regions/update',
-  updateRegionGroups: 'app/regionGroups/update',
+import { getRequestParam } from '@webapp/utils/urlUtils'
+import axios from 'axios'
+import { createI18nPromise } from '@common/i18n/i18nFactory'
+import { applicationError } from '@webapp/components/error/actions'
+
+import * as UserState from '@webapp/store/user/state'
+
+import { sortCountries, sortRegionGroups, sortRegions } from '../utils'
+import ActionTypes from './actionTypes'
+
+export { ActionTypes }
+
+export const updateCountries = (countries) => ({ type: ActionTypes.updateCountries, countries })
+export const updateRegions = (regions) => ({ type: ActionTypes.updateRegions, regions })
+export const updateRegionGroups = (regionGroups) => ({ type: ActionTypes.updateRegionGroups, regionGroups })
+
+export const initApp = () => async (dispatch) => {
+  const lang = getRequestParam('lang')
+  try {
+    const getCountries = axios.get('/api/countries')
+    const getRegions = axios.get('/api/country/regions/')
+    const getRegionGroups = axios.get('/api/country/regionGroups/')
+    const getUserInfo = axios.get(`/api/loggedInUser/`)
```

Comment:
maybe the implementation of axios can be extracted to some wrapper of the api

Reply:
Yes, we had separate pull request/issue for creating a service, this though did not proceed for some reason